cmd/run.go

```diff
@@ -50,8 +50,15 @@ func newRunCmd(rootConfig *RootCommandConfig) *cobra.Command {
 	// runCmd represents the run command
 	var runCmd = &cobra.Command{
 		Use:   "run",
-		Short: "Run the local Appsody environment for your project",
-		Long:  `This starts a docker based continuous build environment for your project.`,
+		Short: "Run your project in the local Appsody environment.",
+		Long: `Run the local Appsody environment, starting a container based continuous build environment for your project.
+		
+You must be in the base directory of your Appsody project when running this command.`,
+		Example: `  appsody run --docker-volume my-volume
+  Runs the local Appsody environment using the "my-volume" docker volume to cache your project dependencies.
+  
+  appsody run --interactive
```

Comment:
Move this before the other example.

Reply:
STDIN -> standard input stream

Reply:
Runs your application in a containerized development environment with attaches the standard input stream to the container. This allows you to interact with the processes within the container.